flipperdevices/flipperzero-firmware
# [FL-2853] Reorganise Universal A/C library

# What's new

- Reorganise Universal remote library for Air Conditioners
- Refactor brute force code

# Verification 

1. Go to `Infrared -> Universal Remotes -> Air Conditioners`
2. Press the buttons. The signals should be sent.
3. Do the same for the TV universal remote for good measure.

# Checklist (For Reviewer)
